tests/validation.rs

```diff
@@ -118,3 +118,20 @@ async fn validate_policy() -> Result<()> {
     assert!(failures.valid(), "manifest is valid");
     Ok(())
 }
+
+#[tokio::test]
+async fn validate_deprecated_configs() -> Result<()> {
+    let (_manifest, failures) = validate_manifest_file(
+        "./tests/fixtures/manifests/deprecated-source-and-target-config.yaml",
+    )
+    .await
+    .context("failed to validate manifest")?;
+    assert!(!failures.valid(), "expected invalid manifest");
```

Comment:
Heads up that I think if all of the failures are warnings, it will be a valid manifest

Reply:
Thanks, I just realized that the manifests link traits target_config and source_config keys are not being stored as values. I might be able to do it when storing the original manifest, which is another issue on the repo. Will look into it early next week. 🙇